js/src/ui/MethodDecoding/methodDecoding.js

```diff
@@ -128,15 +129,25 @@ class MethodDecoding extends Component {
 
   renderMinBlock () {
     const { historic, transaction } = this.props;
-    const { minBlock } = transaction;
+    const { condition } = transaction;
 
-    if (!minBlock || minBlock.eq(0)) {
+    if (!condition) {
       return null;
     }
 
-    return (
-      <span>, { historic ? 'Submitted' : 'Submission' } at block <span className={ styles.highlight }>#{ minBlock.toFormat(0) }</span></span>
-    );
+    if (condition.block && condition.block.gt(0)) {
+      return (
+        <span>, { historic ? 'Submitted' : 'Submission' } at block <span className={ styles.highlight }>#{ condition.block.toFormat(0) }</span></span>
```

Comment:
Historic, didn't want to change - once I do I need to do the whole file, tried to keep changes to the bare minimum.